### DISCUSSION TITLE
Affichage des vidéos dans une chaîne

### DISCUSSION THREAD

**Discussion by DeprezLaurent** [EsupPortail/Esup-Pod#785] [Ideas]
Lorsqu’on visionne une vidéo depuis une chaîne, on a sur la page la description de la chaine et dessous, la vidéo. 
Certaines chaines ont une description assez longue. Du coup il faut scroller pour trouver la vidéo voulue en bas de page. 
Pour éviter cela peut-être faudrait-il, sur la consultation d ‘une vidéo, garder le titre de la chaine et le fil d’Ariane pour ne pas « sortir » de la chaîne, mais ne pas afficher la description ?
Ou réfléchir à un autre affichage avec un bouton « Afficher / masquer la description » ?

Laurent

**Comment by ptitloup** [comment]
Bonjour @DeprezLaurent pourrais-tu nous donner des exemples ? as-tu des urls de vidéos pour lesquels tu souhaitent améliorer l'affichage ? Merci

**Comment by DeprezLaurent** [comment]
Bonjour @ptitloup 
La question m'a été posée pour cette chaine :
https://podeduc.apps.education.fr/devoirs-faits-la-communaute/
J'ai contourné en mettant un accordéon.
Voici un exemple : https://podeduc.apps.education.fr/normandie-college-jean-monnet-flers/musique-et-chant-choral/video/11986-sortie-paris-concert-de-lorchestre-de-paris-a-la-philharmonie-et-exposition-a-la-cite-de-la-musique/
La bannière est très grande + la description + description du thème. Du coup il faut scroller assez loin pour aller voir la vidéo..
Ma première suggestion serait que la description de la chaine ne s'affiche que sur l'accueil de la chaine et celle du thème sur l'accueil du thème,  pas sur la vue de la vidéo...Pour la bannière on peut accompagner les propriétaires pour leur proposer un format plus ergonomique.

**Comment by ptitloup** [comment]
Merci, nous sommes en train de regarder

**Reply by Badatos** [reply]
Le correctif #794 a été mis en oeuvre et sera disponible dans la version 3.1.1 de Pod.

**Comment by ptitloup** [comment]
Les modifications seront donc présents dans la prochaine version de Pod

**Comment by DeprezLaurent** [comment]
Bonjour @ptitloup . excellent ! Du coup passé 140 caractères la description passera en affichage accordéon ?

**Reply by ptitloup** [reply]
oui tout à fait !

**Comment by DeprezLaurent** [comment]
Top ! Merci à vous !
